[NEW] OCA Tests enable

mileo opened the pull request.

mileo committed:
[NEW] OCA Tests enable

[FIX] pep8

[FIX] pep8

[FIX] pep8

rvalyi commented:
:+1: agora qual e o plano fazer o merge e depois acertar os pep8? Tb e possivel excluir algum modulo do Travis se for interessante; nao sei no caso.

mileo commented:
@rvalyi Estamos acerteando os testes pois agora que realmente vamos começar a desenvolver.

rvalyi commented:
beleza

On Mon, Nov 23, 2015 at 5:07 PM, Luis Felipe Miléo <notifications@github.com

> wrote:
> 
> @rvalyi https://github.com/rvalyi Estamos acerteando os testes pois
> agora que realmente vamos começar a desenvolver.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/odoo-brazil/odoo-brazil-banking/pull/23#issuecomment-159030263
> .

## 

Raphaël Valyi
Founder and consultant
http://twitter.com/rvalyi http://twitter.com/#!/rvalyi
+55 21 3942-2434
www.akretion.com

mileo committed:
[FIX] OCA dependencies

[FIX] OCA Dependencies

[NEW] Configure locales and fix cnab dependencies

[FIX] File name

[NEW] PyBoleto add to dependencies

[FIX]

mileo commented:
@rvalyi sabe pq tem 2 testes do travis?

rvalyi commented:
@mileo e porque vc fez o push da branch oca-tests no repo https://github.com/odoo-brazil/odoo-brazil-banking. Se vc tivesse feito no repo https://github.com/kmee/odoo-brazil-banking ai ele teria feito um test so.  Na verdade ele teria feito um nest no push na Kmee e um test aqui na hora de fazer o PR. Nao me pergunte qual e a vantagem dele fazer 2 testes nesse caso. Mas enfim so sei que e por causa disso.

Do restante eu acho que fica melhor deixar as branches dos PR no repo pessoais ou das empresas do que enchendo esse repo de branches. Mas enfim isso e de menos.

mileo committed:
[FIX] Pyboleto

mileo commented:
@fernandomr Tests passing! lets create real case tests and do some refactorys!
cc: @rvalyi @renatonlima @danimaribeiro

danimaribeiro commented:
Qual branch eu testo? cnab

rvalyi commented:
@mileo seria interessante botar os badges Travis e Coveralls no readme.
Tambem, acho que na v8 temos outras prioridades, mas na hora da v9, a gente revisando o modulo de boleto (a parte de persistencia) e atingindo uns 70% de coverage poderemos dar um extract e rebase do modulo para dentro do repo OCA. Por enquanto perfeito se os devs acontecem aqui.

mileo commented:
Eu Vi q tava sem o badge acabei nem comentando nada... 

Vou  trabalhar com outas coisas agora amanhã retomo com o Fernando, estamos fazendo pair programming e acredito q vai ser bem rápido atingir os 70% de coverage.

Abraços

Luís Felipe Mileo

De: Raphaël Valyi notifications@github.com
Enviado: 23/11/2015 7:13 PM
Para: odoo-brazil/odoo-brazil-banking
Cc: Luis Felipe Miléo
Assunto Re: [odoo-brazil-banking] [NEW] OCA Tests enable (#23)

@mileo seria interessante botar os badges Travis e Coveralls no readme.
Tambem, acho que na v8 temos outras prioridades, mas na hora da v9, a gente revisando o modulo de boleto (a parte de persistencia) e atingindo uns 70% de coverage poderemos dar um extract e rebase do modulo para dentro do repo OCA. Por enquanto perfeito se os devs acontecem aqui.

---

Reply to this email directly or view it on GitHub:
https://github.com/odoo-brazil/odoo-brazil-banking/pull/23#issuecomment-159065657

mileo commented:
Danimar,

To bem por fora.... Amanhã te falo, juntei oq o Fernando confirmou q tava bom do Carlos e o lepri e fiz o merge junto com a base de testes.

Amanhã vou fazer um casos de testes e vai dar pra saber melhor a situação.

Crio um novo issue e março vc

Luís Felipe Mileo

De: Danimar Ribeiro notifications@github.com
Enviado: 23/11/2015 7:11 PM
Para: odoo-brazil/odoo-brazil-banking
Cc: Luis Felipe Miléo
Assunto Re: [odoo-brazil-banking] [NEW] OCA Tests enable (#23)

Qual branch eu testo? cnab

---

Reply to this email directly or view it on GitHub:
https://github.com/odoo-brazil/odoo-brazil-banking/pull/23#issuecomment-159065335
